FULL_HELP_DOCS.md

```diff
@@ -1802,6 +1747,89 @@ Allows issuing account to configure authorization and trustline flags to an asse
 
 
 
+## `stellar tx send`
+
+Send a transaction envelope to the network
+
+**Usage:** `stellar tx send [OPTIONS]`
+
+###### **Options:**
+
+* `--rpc-url <RPC_URL>` — RPC server endpoint
+* `--rpc-header <RPC_HEADERS>` — RPC Header(s) to include in requests to the RPC provider
+* `--network-passphrase <NETWORK_PASSPHRASE>` — Network passphrase to sign the transaction sent to the rpc server
+* `--network <NETWORK>` — Name of network to use from config
+* `--global` — Use global config
+* `--config-dir <CONFIG_DIR>` — Location of config directory, default is "."
+
+
+
+## `stellar tx set`
+
+Set various options for a transaction
+
+**Usage:** `stellar tx set [OPTIONS]`
+
+###### **Options:**
+
+* `--sequence-number <SEQUENCE_NUMBER>` — Set the transactions sequence number
+* `--fee <FEE>` — Set the transactions fee
+* `--memo-text <MEMO_TEXT>` — Set the transactions memo text
+* `--memo-id <MEMO_ID>` — Set the transactions memo id
+* `--memo-hash <MEMO_HASH>` — Set the transactions memo hash
+* `--memo-return <MEMO_RETURN>` — Set the transactions memo return
+* `--source-account <SOURCE_ACCOUNT>` — Change the source account for the transaction
+* `--max-time-bound <MAX_TIME_BOUND>` — Set the transactions max time bound
+* `--min-time-bound <MIN_TIME_BOUND>` — Set the transactions min time bound
+* `--min-ledger <MIN_LEDGER>` — Set the minimum ledger that the transaction is valid
+* `--max-ledger <MAX_LEDGER>` — Set the max ledger that the transaction is valid. 0 or not present means to max
+* `--min-seq-num <MIN_SEQ_NUM>` — set mimimum sequence number
+* `--min-seq-age <MIN_SEQ_AGE>`
+* `--min-seq-ledger-gap <MIN_SEQ_LEDGER_GAP>` — min sequeence ledger gap
+* `--extra-signers <EXTRA_SIGNERS>` — Extra signers
+* `--no-preconditions` — Set precondition to None
```

Comment:
Playing around with this interface I think some design discussion is required for this sub-command.

When we previously discussed this (https://github.com/stellar/stellar-cli/issues/1643#issuecomment-2459724928) I had suggested we could have one sub-command per field to set, but I think this arrangement is also reasonable, but looks like there's missing functionality as it is, so we need that design to flesh out all the use cases.

For example, it looks like we can clear all the preconditions with 'no-preconditions', but it's not obvious to a user which fields are being cleared because the options are bundled in a single command the relationship between all the options appear equal / related, when they are not.

Another example, there's no way to remove a memo, or some precondition fields without clearing all preconditions.

I've started a design discussion on the issue:
- https://github.com/stellar/stellar-cli/issues/1643#issuecomment-2510447038